src/ExpressModuleConfig.ts

```diff
@@ -26,6 +26,59 @@ export interface ExpressModuleConfig {
     };
 
     /**
+     * Options for compression middleware.
+     *
+     * See https://github.com/expressjs/compression for more information.
+     */
+    compression?: {
+
+        /**
+         * If set to true, compression middleware will be enabled.
+         */
+        enabled?: boolean;
```

Comment:
maybe this should be required option? Because otherwise neither of options are useful